### ISSUE TITLE
'Alt' modifier not detected in some environments breaks PointManipulator

### ISSUE THREAD

**Issue by alisterburt** [napari-threedee/napari-threedee#94]
napari/napari#5612

Possible workaround is to replace the current logic for early exit with

```
    # Early exit if not alt-clicked
    if ('Alt' not in event.modifiers) and ('Control' not in event.modifiers):
        return
```

**Comment by psobolewskiPhD** [comment]
Did you mean annotator here, not manipulator? I don't think `alt` is used with manipulators?
If so, do you think this closes: https://github.com/napari-threedee/napari-threedee/pull/227 ?

**Comment by alisterburt** [comment]
Yep, closed by #227
